.github/workflows/lint.yml

```diff
@@ -7,15 +7,47 @@ on:
       - develop
 
 jobs:
-  formatting:
+  formatting-juia:
+    if: "!contains(github.event.head_commit.message, 'skip_ci')"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.3.0]
+        julia-arch: [x86]
+        os: [ubuntu-latest]
```

Comment:
Looks like `julia-arch` isn't used anywhere. I'd say we can also just hard code the `runs-on: ubuntu-latest` (we probably won't be running formatting on multiple OSes).

Reply:
Done.